blaze/compute/tests/test_url_csv_compute.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+import os
+
+from blaze import Data, compute
+from blaze.utils import raises
+from odo import URL, CSV
+
+import pandas as pd
+import pandas.util.testing as tm
+
+from functools import partial
+
+try:
+    from urllib2 import urlopen
+    from urllib2 import HTTPError, URLError
+except ImportError:
+    from urllib.request import urlopen
+    from urllib.error import HTTPError, URLError
+
+pytestmark = pytest.mark.skipif(raises(URLError,
+                                       partial(urlopen, "http://google.com")),
+                                reason='unable to connect to google.com')
+
+iris_url = ('https://raw.githubusercontent.com/'
```

Comment:
Could we have the test spin up a local webserver and then serve this test data? I would prefer to be able to run the tests without internet connection.